mkosi.postinst.chroot

```diff
@@ -2,6 +2,12 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 set -e
 
+# These workarounds are for Fedora, and break logins in Debian/Ubuntu
+source /etc/os-release
+if [ "$ID" = "debian" ] || [ "$ID_LIKE" = "debian" ]; then
+    exit 0
+fi
```

Comment:
I don't think these workaround break login on Debian/Ubuntu as long as we make a symlink from /etc/pam.d to /usr/lib/pam.d with tmpfiles, so please also drop this.